app/views/static/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= javascript_pack_tag 'application' %>
+
+<%= react_component("App") %>
```

Comment:
- [ ] the view should only have a div with a class called root, the rest of the application should be rendered by react.